src/seismometer/plot/mpl/binary_classifier.py

```diff
@@ -396,3 +397,33 @@ def performance_metrics(
     if highlight is not None:
         lines.vertical_threshold_lines(axis, highlight, color_alerts=True)
     return axis.get_figure()
+
+
+@export
+@model_plot
+def plot_metric_list(
+    stats: pd.DataFrame,
+    metrics: list[str],
+) -> plt.Figure:
+    """
+    Plots a list of metrics vs threshold.
+
+    Parameters
+    ----------
+    stats : pd.DataFrame
+        The table of performance metrics with the index being threshold percentiles.
+    metrics : list[str]
+        The performance metrics to plot, must be columns in the stats dataframe.
+    """
+    fig = plt.figure(figsize=(6, 4))
+    axis = fig.gca()
+
+    for metric in metrics:
+        thresholds = as_probabilities(stats.index)
+        axis.plot(thresholds, stats[metric], label=metric)
+        axis.legend(loc="lower right")
+
+        axis.set_xlim([0, 1.01])
+        axis.set_xlabel("Threshold")
```

Comment:
Maybe move to outside of the loop. (Also, line 422)

Reply:
```suggestion
        axis.legend(loc="lower right")

        axis.set_xlim([0, 1.01])
    axis.set_xlabel("Threshold")
```